result/src/iosMain/kotlin/com/github/kittinunf/result/runCatching.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.kittinunf.result
+
+actual inline fun <V> runCatching(block: () -> V?): Result<V?, Throwable> {
+    return try {
```

Comment:
Then we can use the new `doTry` function and there's no need for expect actual dance right?

Reply:
I am afraid that be impossible because it expects receivers.

Reply:
I made PR on top of yours #104